test/util/report.js

```diff
@@ -277,14 +277,17 @@ fs.readdirSync(testsPath).forEach(function (testFile) {
   }
 });
 
-console.log('\nExamples without any regression tests:\n');
+console.log('\n#### Regression test coverage:\n');
+console.log('\n#### Examples without any regression tests:\n');
 console.log(examplesWithNoTestsReport || 'None found.\n');
-console.log('\nExamples missing some regression tests:\n');
+console.log('\n#### Examples missing some regression tests:\n');
 console.log(examplesMissingSomeTestsReport || 'None found.\n');
-console.log('\nExamples documentation table rows without data-test-ids:\n');
+console.log(
+  '\n#### Examples pages Keyboard or Attribute table rows that do not have data-test-ids:\n'
```

Comment:
Thanks @spectranaut ! That's much clearer. Just one teeny tiny thing more...

"Examples pages" doesn't quite sound right - could we say either:
- "Example pages with"
or
- "Examples with"

```suggestion
  '\n#### Examples with Keyboard or Attribute table rows that do not have data-test-ids:\n'
```

Reply:
good catch :) fixed! to "pages with"